src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -451,7 +451,8 @@ private OpenApiOperation CreateBaseOperation(string description, List<OpenApiTag
             if (dbObject.SourceType == EntitySourceType.StoredProcedure)
             {
                 Entity entityTest = _runtimeConfig.Entities[entityName];
-                List<SupportedHttpVerb> spRestMethods = entityTest.Rest.Methods.ToList();
+
+                List<SupportedHttpVerb>? spRestMethods = (entityTest.Rest.Methods is not null) ? entityTest.Rest.Methods.ToList() : null;
```

Comment:
Shouldn't we populate this list spRestMethods with POST?

Reply:
updated